sdk/python/flet/tooltip.py

```diff
@@ -84,10 +85,10 @@ def __init__(
         gradient: Optional[Gradient] = None,
         border: Optional[Border] = None,
         border_radius: BorderRadiusValue = None,
-        shape: Optional[BoxShape] = None,
+        shape: Union[BoxShape, BoxShapeString] = None,
```

Comment:
This shows error:

![image](https://user-images.githubusercontent.com/5041459/206920596-1c28c93f-982b-43e0-ba47-ec81eabbb1fa.png)